apps/exam/views.py

```diff
@@ -46,33 +46,55 @@ def get_context_data(self, **kwargs):
     """Render intro page template for chosen exam. """
 
     context = super().get_context_data(**kwargs)
-    exam = Exam.objects.get(id=self.kwargs['exam_id'])
-    context['exam'] = exam
-    context[
-        'attempt_duration'] = exam.duration_minutes + self.request.user.required_extra_time
+    if 'exam_id' in self.kwargs:
+      exam = Exam.objects.filter(id=self.kwargs['exam_id'])
```

Comment:
Here you have just a single exam which can be retrieved using .get()

Reply:
Again, I need a list of exams, even if it's just one.